tests/FunctionsTest.php

```diff
@@ -98,4 +130,118 @@ public function testIterableKeyFirstReturnsNullIfIterableIsGeneratorAndIsEmpty()
             yield;
         })));
     }
+
+    public function testYieldGroupsWithEmptyIterator()
+    {
+        $this->assertEquals([], iterator_to_array(Stdlib\yield_groups(new EmptyIterator(), function () {
+        })));
+    }
+
+    public function testYieldGroupsCallbackArguments()
+    {
+        Stdlib\yield_groups(new ArrayIterator(static::YIELD_GROUPS_DATA), function (array $v, string $k): int {
+            $this->assertEquals(static::YIELD_GROUPS_DATA[$k], $v);
+
+            return $v['group'];
+        });
```

Comment:
Forgot to assert that :)

Reply:
No. This test is about the callback arguments. Not the return.

Reply:
The assertion inside the callback is never run and I don't know how the GitHub actions even succeeded.

Reply:
That is so true.

Reply:
```bash
This test did not perform any assertions

/usr/local/src/ipl-stdlib/tests/FunctionsTest.php:140
```